server/config/fusor.yaml

```diff
@@ -228,7 +240,7 @@
            :name: "OpenShift"
            :content: :openshift
            :subscription_descriptions:
-             - "Red Hat Cloud Infrastructure"
+             - "Red Hat OpenShift Enterprise"
```

Comment:
@johnwmatthews it might be correct for now. It will change to rhci down the road. But I think they are separate for now. I'll have to confirm in the activation key code